README.md

```diff
@@ -23,17 +22,19 @@ REACT_APP_ETHERSCAN_URL=      the etherscan url (https://etherscan.io, etc.)
 REACT_APP_SYNC_INTERVAL=      max seconds watcher has to sync to ethereum before blocking further transactions
 REACT_APP_POLL_INTERVAL=      number of seconds to poll account data
 REACT_APP_NETWORK=            the network your environment is on (ropsten, rinkeby, private, main, etc.)
+REACT_APP_ALTERNATE_WALLETS=  *optional* other web wallets hosted on different envs, follow this format -> name,url|name,url|name,url
```

Comment:
Have a doubt on the optionals here. The app crashes if `REACT_APP_ALTERNATE_WALLETS` is blank. Also presume WalletConnect does not work without the proxy.

Reply:
ok nice catch, fixed the crash on no alternates, and yea wont work without the proxy but the ui should already detect that and disable the walletconnect/link options automatically. checking about the sai bug now